components/server/src/oauth-server/db.ts

```diff
@@ -141,6 +141,17 @@ const desktopClient: OAuthClient = {
     ],
 };
 
+const toolbox: OAuthClient = {
+    id: "toolbox-gateway-gitpod-plugin",
+    name: "JetBrains Toolbox Gitpod Plugin",
+    redirectUris: ["jetbrains://gateway/io.gitpod.toolbox.gateway/auth"],
+    allowedGrants: ["authorization_code"],
+    scopes: [
+        // We scope all so that it can work in papi like a PAT
+        { name: "function:*" },
```

Comment:
cc @geropl @svenefftinge it is because we did not finish tokens for desktop clients for public API